home.admin/config.scripts/lnd.install.sh

```diff
@@ -4,7 +4,7 @@
 ## based on https://raspibolt.github.io/raspibolt/raspibolt_40_lnd.html#lightning-lnd
 ## see LND releases: https://github.com/lightningnetwork/lnd/releases
 ### If you change here - make sure to also change interims version in lnd.update.sh #!
-lndVersion="0.16.4-beta"
+lndVersion="0.17.1-beta"
```

Comment:
There has been  a quick bugfix release:
https://github.com/lightningnetwork/lnd/releases/tag/v0.17.2-beta
```suggestion
lndVersion="0.17.2-beta"
```